src/Tribe/Views/V2/Template/JSON_LD.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Handles JSON-LD for V2.
+ *
+ * @since TBD
+ *
+ * @package Tribe\Events\Views\V2\Template
+ */
+
+namespace Tribe\Events\Views\V2\Template;
+
+/**
+ * Class JSON_LD
+ *
+ * @since TBD
+ *
+ * @package Tribe\Events\Views\V2\Template
+ */
+class JSON_LD {
+
+	/**
+	 * Fires to add JSON LD to Single Event
+	 *
+	 * @since TBD
+	 *
+	 */
+	public function print_single_json_ld() {
```

Comment:
:no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: 3662be4d7cc7e824f9308d885bdfe590</sub>